lisp/pdf-annot.el

```diff
@@ -1509,6 +1509,25 @@ annotation's contents and otherwise `text-mode'. "
   :group 'pdf-annot
   :type display-buffer--action-custom-type)
 
+(defcustom pdf-annot-list-format
+  '((page . 3)
+    (type . 10)
+    (label . 24)
+    (date . 24))
+    "Annotation properties visible in the annotation list."
+  :type '(alist :key-type (symbol))
+  :options '((page (integer :value 3 :tag "Column Width"))
+	     (type (integer :value 10 :tag "Column Width" ))
+	     (label (integer :value 24 :tag "Column Width"))
+	     (date (integer :value 24 :tag "Column Width"))
+	     (comment (integer :value 56 :tag "Column Width")))
```

Comment:
Did you mean to write `contents` ?

Reply:
I think of annotations as having comments, but `contents` would probably be more accurate.
```suggestion
	     (contents (integer :value 56 :tag "Column Width")))
```

Reply:
I see this marked as "suggested change", but I don't see a button to commit it.

Reply:
I've added it as a commit